draft-ietf-taps-interface.md

```diff
@@ -1933,34 +1936,35 @@ and remove local endpoints to/from a Connection.
 
 # Managing Connections {#introspection}
 
-During pre-establishment and after establishment, connections can be configured and queried using Connection
+During pre-establishment and after establishment, (Pre-)Connections can be configured and queried using Connection
 Properties, and asynchronous information may be available about the state of the
-connection via Soft Errors.
+Connection via Soft Errors.
 
 Connection Properties represent the configuration and state of the selected
 Protocol Stack(s) backing a Connection. These Connection Properties may be
 Generic, applying regardless of transport protocol, or Specific, applicable to a
-single implementation of a single transport protocol stack. Generic Connection
+single implementation of a single transport Protocol Stack. Generic Connection
 Properties are defined in {{connection-props}} below.
 
 Protocol-specific Properties are defined in a transport- and
 implementation-specific way to
 permit more specialized protocol features to be used.
 Too much reliance by an application on Protocol-specific Properties can significantly reduce the flexibility
-of a transport services implementation to make appropriate
+of a transport services system to make appropriate
 selection and configuration choices. Therefore, it is RECOMMENDED that
 Protocol-specific Properties are used for properties common across different protocols and that
 Protocol-specific Properties are only used where specific protocols or properties are necessary.
 
 The application can set and query Connection Properties on a per-Connection
 basis. Connection Properties that are not read-only can be set during
-pre-establishment (see {{selection-props}}), as well as on connections directly using
+pre-establishment (see {{selection-props}}), as well as on Connections directly using
 the SetProperty action:
 
 ~~~
-Connection.SetProperty(property, value)
+ErrorCode := Connection.SetProperty(property, value)
 ~~~
 
+If an error is encountered in setting a property (for example, if the application tries to set a TCP-specific property on a Connection that is not using TCP), the application MUST be informed about this error via the `ErrorCode` Object. Such errors MUST NOT cause the Connection to be terminated.
```

Comment:
That's the one item that truly addresses #1131. I think this was important to include here.